doc/design/cluster_train/submit-job.md

```diff
@@ -0,0 +1,115 @@
+# Submit a Distributed Training Job
+
+If a user wants to start up a distributed training job, he will submit the distributed training job with Python code.
```

Comment:
文章题目是"Submit a Distributed Training Job"，是不是本地训练就不用说了（或者不用详细说，这里本地训练的介绍字数比远程训练的介绍字数还多）。
是不是可以改成：
The user can submit a distributed training job with Python code, rather than with a command-line interface.